layers/error_message/logging.cpp

```diff
@@ -315,7 +314,7 @@ std::string DebugReport::CreateMessageText(VkFlags msg_flags, const Location &lo
         uint32_t num_vuids = sizeof(vuid_spec_text) / sizeof(vuid_spec_text_pair);
         const char *spec_text = nullptr;
         // Only the Antora site will make use of the sections
-        std::string spec_url_section;
+        const char *spec_url_section = nullptr;
```

Comment:
```suggestion
        const char *spec_url_section = "";
```
This will not get assigned for UNASSIGNED vuids for instance, so need to prevent nullptr dereference

Reply:
it will never make it here, we wrap this with `if ((vuid_text.find("VUID-") != std::string::npos)) {`

Reply:
but in the case we can't find the VUID, agree this is bad to be null, I will keep as `nullptr` and do a check to skip the whole "construct the spec link" code if `spec_url_section` is null

Reply:
now do a `if (spec_text && spec_url_section) {` check